addons/test_mail/tests/test_message_management.py

```diff
@@ -27,6 +27,7 @@ def setUpClass(cls):
             'name': 'Partner 2',
             'email': 'p2@example.com'
         })
+        cls.user_partner = cls.env.user.partner_id
```

Comment:
Don't need this indirection, setup is for actually creating stuff, here the user is already existing.